cogs/help.py

```diff
@@ -40,11 +40,17 @@ async def on_message(self, message):
         if message.channel.id == self.help_channel and message.type == MessageType.thread_created:
             await message.delete(delay=5)
 
-
     @commands.command()
     @commands.is_owner()
     async def help_menu(self, ctx):
         await ctx.send("Click a button to create a help thread!", view=HelpView())
 
+    @commands.command()
+    @commands.has_permissions(manage_messages=True)
```

Comment:
This should be limited to thread creator and helpers.

Reply:
The thread creator is always the bot, I can make it helpers only though.

Reply:
You should be able to determine who requested the thread to be opened. Worst case you can check the `mentions` in the first message.

Reply:
Fixed in 5b52730.